app/Jobs/DeleteQueryserviceNamespaceJob.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace App\Jobs;
+
+use App\QueryserviceNamespace;
+use App\Http\Curl\CurlRequest;
+use App\Http\Curl\HttpRequest;
+use Illuminate\Contracts\Queue\ShouldBeUnique;
+
+class DeleteQueryserviceNamespaceJob extends Job implements ShouldBeUnique
```

Comment:
At some point I think it will make sense to group these jobs a bit more, for example we have a provision and a delete for query service namespaces.
Similar for elastic search and wiki dbs etc.

Reply:
Yeah for sure, they should end up in separate namespaces when everything is merged